trino/trino.go

```
@@ -1326,12 +1726,39 @@ func (qr *driverRows) fetch() error {
 			if qresp.ID == "" {
 				return io.EOF
 			}
+
 			err = qr.initColumns(&qresp)
 			if err != nil {
 				return err
 			}
+
 			qr.rowindex = 0
-			qr.data = qresp.Data
+			switch data := qresp.Data.(type) {
+			case []interface{}:
+				// direct protocol
+				qr.data = make([]queryData, len(data))
+				for i, item := range data {
+					if row, ok := item.([]interface{}); ok {
```

Should we throw an error if the type assertion fails?